boards/arm/particle_xenon/dts/mesh_feather_uart2_rtscts.dtsi

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2020 Peter Bigot Consulting, LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Add hardware flow control to UART2, which is exposed only on Xenon
+ * mesh boards.
+ */
+&uart1 {
+	rts-pin = <35>;
+	cts-pin = <33>;
```

Comment:
Shouldn't it be 43 (P1.11)?

Reply:
You're correct; will update.

Reply:
Done; (had to rebase for buildkite).

Thanks for catching this.  Another example of why https://github.com/zephyrproject-rtos/zephyr/pull/23728#issuecomment-603790669 would benefit from some attention soon: the offset integral pin representation is opaque and error-prone.